tests/tests_stratis.yml

```diff
@@ -248,31 +257,34 @@
                 nbde_server_manage_selinux: true
                 nbde_server_port: 7500
 
-            - name: Create encrypted Stratis pool with Clevis/Tang
-              include_role:
-                name: linux-system-roles.storage
-              vars:
-                storage_pools:
-                  - name: foo
-                    disks: "{{ unused_disks[0] }}"
-                    type: stratis
-                    encryption: true
-                    encryption_password: yabbadabbadoo
-                    encryption_clevis_pin: tang
-                    encryption_tang_url: localhost:7500
-
-            - name: Verify role results
-              include_tasks: verify-role-results.yml
-
-            - name: Clean up
-              include_role:
-                name: linux-system-roles.storage
-              vars:
-                storage_pools:
-                  - name: foo
-                    disks: "{{ unused_disks[0] }}"
-                    type: stratis
-                    state: absent
+            - name: Clevis/Tang test
+              block:
+                - name: Create encrypted Stratis pool with Clevis/Tang
+                  include_role:
+                    name: linux-system-roles.storage
+                  vars:
+                    storage_pools:
+                      - name: foo
+                        disks: "{{ unused_disks[0] }}"
+                        type: stratis
+                        encryption: true
+                        encryption_password: yabbadabbadoo
+                        encryption_clevis_pin: tang
+                        encryption_tang_url: localhost:7500
+
+                - name: Verify role results
+                  include_tasks: verify-role-results.yml
+
+              always:
```

Comment:
I guess we should add this style of cleanup to all tests, but that's definitely out of scope for this PR.